src/v2/coins/bsv.js

```diff
@@ -24,6 +24,10 @@ class Bsv extends Bch {
     return 'Bitcoin SV';
   }
 
+  getAddressPrefix() {
+    return 'bitcoinsv';
```

Comment:
I don't think they changed the bech32 address prefix, and if they did, this is may be a problem when reading old BCH addresses.

Reply:
Removed.